### PR TITLE
Add welcome view for new contributors

### PR DESCRIPTION
**Pull request by tgodzik**
I tried to add a variety of commands that I used that could also help people new to sbt overall.

Selection_042

### CODE REVIEW

**Comment by tgodzik** [comment]
I am not tied to the ASCII art here, if anyone finds something nicer, let me know! I used http://patorjk.com/software/taag/#p=display&f=Star%20Wars&t=Metals

**Comment by ckipp01** [comment]
> I am not tied to the ASCII art here, if anyone finds something nicer, let me know! I used http://patorjk.com/software/taag/#p=display&f=Star%20Wars&t=Metals

```
             _        _     
  /\/\   ___| |_ __ _| |___ 
 /    \ / _ \ __/ _` | / __|
/ /\/\ \  __/ || (_| | \__ \
\/    \/\___|\__\__,_|_|___/
                            
```

I'm more inclined to have something more *metal*!

File: docs/contributors/getting-started.md [hunk 1]
```diff
@@ -77,16 +77,16 @@ To run the unit tests open an sbt shell and run `unit/test`
...
+> unit/testOnly tests.DefinitionSuite
 # run a specific test case inside the suite.
-> metals/testOnly tests.DefinitionSuite -- *exact-test-name*
+> unit/testOnly tests.DefinitionSuite -- *exact-test-name*
 # run unit tests, moderately fast but still a bit too slow for edit/test/debug workflows.
 > unit/test
 # run slow integration tests, takes several minutes.
 > slow/test
-# run slow presentation compiler tests
+# run presentation compiler tests, these are the quickest tests to run.
```
Comment (ckipp01) [inline docs/contributors/getting-started.md:87]: Does anyone else find it ironic that we say our tests in `slow` are the quickets tests to run?

**Review by ckipp01 — COMMENTED** [review]
This is a nice touch. 👍 

I've forgotten about this, but one thing I've been meaning to do, which would be nice here is to also explain how to publish something when working on Scala 3 features. It may not be super obvious and people may end up trying to do a fully `+publishLocal` instead of just what they need for the Scala3 version of metags that they really want. Should we maybe add something along the lines of "If you're working on a a Scala 3 feature, here is the best way to build to test this" or something?

EDIT:.... I realize I wrote metegs because I was thinking metals and was writing mtags. Did I just come up with a great new name or what, metags!

File: docs/contributors/getting-started.md (hunk 1 above)
Comment (tgodzik) [reply docs/contributors/getting-started.md:87]: I think that comment was wrong really, it's for the cross test, which are the quickest to run, not sure why we said `slow` before :O

File: project/Welcome.scala [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import sbtwelcome._
+
+object Welcome {
+
+  val logo =
+    """|             _        _
```
Comment (tgodzik) [inline project/Welcome.scala:6]: Changed to a more metals feel :metal:

File: project/Welcome.scala [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      "publish Metals for the currently used Scala version."
+    ),
+    UsefulTask(
+      "",
+      "+publishLocal",
+      "publish Metals for all supported used Scala versions."
+    ),
+    UsefulTask(
+      "",
+      "++3.0.0-M2 mtags/publishLocal",
```
Comment (tgodzik) [inline project/Welcome.scala:52]: @ckipp01 I added the additional command for publishing a single version.

File: project/Welcome.scala (hunk 2 above)
Comment (ckipp01) [reply project/Welcome.scala:6]: ⚡

**Review by ckipp01 — APPROVED** [review]
Looks good to me. LGTM!

**Review by tanishiking — APPROVED** [review]
LGTM!
It's lovely! and mentioning about publishing mtags3 to local is definitely useful for new contributors (because I actually had a bit trouble when I contribute to metals for the first time in a while) 👍

**Pull request merged**